.travis.yml

```diff
@@ -14,62 +15,70 @@ script:
  - cd $ROLE && ../scripts/test.sh
 
 env:
- - ROLE=ansible-role-analysis-tools SCENARIO=
- - ROLE=ansible-role-anonymous-ftp SCENARIO=
- - ROLE=ansible-role-basedeps SCENARIO=
- - ROLE=ansible-role-cadvisor SCENARIO=
- - ROLE=ansible-role-cli-utils SCENARIO=
- - ROLE=ansible-role-deploy-archive SCENARIO=
+ - ROLE=ansible-role-analysis-tools SCENARIO=default
+ - ROLE=ansible-role-anonymous-ftp SCENARIO=default
+ - ROLE=ansible-role-basedeps SCENARIO=default
+ - ROLE=ansible-role-cadvisor SCENARIO=default
+ - ROLE=ansible-role-cli-utils SCENARIO=default
+ - ROLE=ansible-role-debug-dumpallvars SCENARIO=
+ - ROLE=ansible-role-deploy-archive SCENARIO=default
+ - ROLE=ansible-role-docker SCENARIO=default
  - ROLE=ansible-role-docker-tools SCENARIO=
- - ROLE=ansible-role-fluentd SCENARIO=
- - ROLE=ansible-role-hosts-populate SCENARIO=
+ - ROLE=ansible-role-fluentd SCENARIO=default
+ - ROLE=ansible-role-hosts-populate SCENARIO=default
+ - ROLE=ansible-role-ice SCENARIO=ice36-ubuntu1804
  - ROLE=ansible-role-ice SCENARIO=ice36all
  - ROLE=ansible-role-ice SCENARIO=ice36minimal
- - ROLE=ansible-role-ice SCENARIO=ice36-ubuntu1804
- - ROLE=ansible-role-iptables-raw SCENARIO=
+ - ROLE=ansible-role-iptables-raw SCENARIO=default
  - ROLE=ansible-role-java SCENARIO=centos
  - ROLE=ansible-role-java SCENARIO=ubuntu
- - ROLE=ansible-role-local-accounts SCENARIO=
+ - ROLE=ansible-role-local-accounts SCENARIO=default
  - ROLE=ansible-role-logrotate SCENARIO=
  - ROLE=ansible-role-lvm-partition SCENARIO=
- - ROLE=ansible-role-mysql-backup SCENARIO=
- - ROLE=ansible-role-network-cloud-interfaces SCENARIO=
```

Comment:
https://github.com/ome/ansible-role-munin is missing from this list. I can't see anything that would cause it to be skipped by the script.

Reply:
Thanks. I noticed that the GitHub search API does not include forks by default. With 
10e7a5e, `.travis.yml` was regenerated and includes a few forked roles.